### PR TITLE
improve style of "add repository" dialogs

### LINKED ISSUE #887: improve style of "add repository" dialogs
see #886

### PR DESCRIPTION
**Pull request by plata**
fixes #887

type
local
git
classpath

### CODE REVIEW

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/RepositoriesPanel.java [hunk 1]
```diff
@@ -127,6 +127,7 @@ private void populateRepositoryGrid() {
         this.addButton.setText(tr("Add"));
         this.addButton.setOnAction((ActionEvent event) -> {
             AddRepositoryDialog dialog = new AddRepositoryDialog();
+            dialog.initOwner(this.getParent().getScene().getWindow());
```
Comment (madoar) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/RepositoriesPanel.java]: Just asking:
What does this line do?
Does it add the stylesheet from the parent to the dialog?

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/addrepository/ChooseRepositoryTypePanel.java [hunk 2]
```diff
@@ -74,6 +75,8 @@ public RepositoryType fromString(String string) {
         choiceBoxLabel.setLabelFor(choiceBox);
 
         HBox content = new HBox(choiceBoxLabel, choiceBox);
+        content.setAlignment(Pos.CENTER_LEFT);
```
Comment (madoar) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/addrepository/ChooseRepositoryTypePanel.java]: I don't like this approach.
I think we should set these things via css.

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/RepositoriesPanel.java (hunk 1 above)
Comment (plata) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/RepositoriesPanel.java]: Didn't check if the stylesheet is applied but if you do not have this (probably yes), the dialog does not use the correct window icon.

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/addrepository/ChooseRepositoryTypePanel.java (hunk 2 above)
Comment (plata) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/addrepository/ChooseRepositoryTypePanel.java]: I'm not sure. Do we really want every little dialog to be themeable?

Comment (madoar) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/addrepository/ChooseRepositoryTypePanel.java]: It's not only a question about themeability. 
The important benefit of moving **every** style relevant line of code to css is, that we have everything in a single place, in a single file.
This in turn makes it quite easy to change some/fix some wrong layout without the need to search the whole codebase first.

In addition:
I don't think it's bad if make the dialogs themeable. It doesn't give us any disadvantage I think, right?

**Pull request merged**